php/commands/auth.php

```diff
@@ -65,6 +60,12 @@ public function login($args, $assoc_args) {
       }
 
       $this->auth->logInViaUsernameAndPassword($email, $password);
+    } else {
+      $token = '';
+      if (isset($assoc_args['session'])) {
+        $token = $assoc_args['session'];
+      }
+      $this->auth->logInViaSessionToken($token);
```

Comment:
I think we'll be able to get rid of this altogether with this release. Auth0 refresh tokens will work for Pantheors so there'll be no need for session login.